roles/httpd/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: install httpd
+  yum:
+      name: httpd
```

Comment:
@larsks: I indeed meant that :)

@mrunge: Having package/service/config/etc. configurable does not necessarily means we are targeting for multiple distributions (although it makes it easier). I'm just trying to create stuff which is easy to change in case it is required in the future.